README.md

```diff
@@ -142,7 +144,7 @@ waterwheel.api.user.get(1)
   });
 ```
 `.get()` accepts two arguments
-  - `token`: The token of the entity you are requesting, `nid`.
+  - `identifier`: The identifier of the entity you are requesting, `nid`.
   - `type`: The type of response you are requesting. Currently this is optional, and internally defaults to `JSON`.
```

Comment:
Why use "type" here instead of "format"? Type could be confused with entity type.